include/aspect/mesh_deformation/interface.h

```diff
@@ -157,6 +170,11 @@ namespace aspect
          */
         void update();
 
+        /**
+         * Compute the initial mesh deformation from plugins that implement it.
+         */
+        void execute_initial_deformation();
```

Comment:
Does it *compute* it, or does it actually *execute* the deformation? Function name and documentation disagree :-)

(Separately, maybe "perform" is a better word than "execute".)